code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -53,3 +53,26 @@
 /obj/item/weapon/melee/chainsword/suicide_act(mob/user)
 	viewers(user) << "\red <b>[user] is slicing \himself apart with the [src.name]! It looks like \he's trying to commit suicide.</b>"
 	return (BRUTELOSS|OXYLOSS)
+
+
+//This is essientially a crowbar and a baseball bat in one.
+/obj/item/weapon/melee/misc/hammer
+	color
+	name = "kneebreaker hammer"
+	desc = "An intimidating hammer made of plasteel, this would hurt."
+	icon = 'icons/obj/kneehammer.dmi'
+	icon_state = "kneehammer"
+	item_state = "kneehammer"
+	contained_sprite = 1
+	slot_flags = SLOT_BELT
+	force = 22
+	throwforce = 70

```

Comment:
You would think, but in my own testing 22 throw force does scratch damage in the single-digit range, while 70 changes between 13 to 17 brute.